model.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, sess, image_size=64, is_crop=False,
 
         # batch normalization : deals with poor initialization helps gradient flow
         self.d_bns = [
-            batch_norm(name='d_bn{}'.format(i,)) for i in range(4)]
+            batch_norm(name='d_bn{}'.format(i+1,)) for i in range(3)]
```

Comment:
Only d_bns[0..2] are used. They are named d_bn1, d_bn2 and d_bn3 in the current saved checkpoint, so need to keep those names for it to load.